src/ts/greedy-nav/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why do we need a separate tsconfig file in here? Can the top level one not deal with it?

Reply:
The top level one sets up the sub projects, and there's also a base config that this one extends.

we'll have one tsconfig.json per component - generally they'll just do what this one does - says "build everything to one file" but we also have the option to output to multiple files or override specific configuration if we need to.

The short answer is that this is the mechanism that allows us to build each component to a separate file. It uses a newish feature called Project References - make sense?

Reply:
👍